### PR TITLE
[PLAY-1084] Linting K-O

### PR DESCRIPTION
**Pull request by jasperfurniss**
**What does this PR do?** A clear and concise description with your runway ticket url.

**Screenshots:** Screenshots to visualize your addition/change

**How to test?** Steps to confirm the desired behavior:
4. See addition/change

#### Checklist:
- [X] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [X] **DEPLOY** I have added the `milano` label to show I'm ready for a review.
~- [ ] **TESTS** I have added test coverage to my code.~

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_label_value/_label_value.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ type LabelValueProps = {
   aria?: { [key: string]: string };
   className?: string;
   dark?: boolean;
-  data?: object;
+  data?: Record<string, unknown>;
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_label_value/_label_value.tsx:18]: There is a `GenericObject` type in my commit [here](https://github.com/powerhome/playbook/pull/3021/commits/e752e034e34b62e0b7e9909445463447f1ed5a9e) if you want to cherry pick and use that instead

File: playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx [hunk 2]
```diff
@@ -20,8 +20,8 @@ type CarouselType = {
 export default function Carousel({
   currentIndex,
   photos,
-  onClick = ()=>{},
-  onChange = ()=>{},
+  onClick,
+  onChange,
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx:24]: You can also do:

```tsx
onClick = () => undefined,
```

for these which would be more consistent as a void function.

File: playbook/app/pb_kits/playbook/pb_label_value/_label_value.tsx (hunk 1 above)
Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_label_value/_label_value.tsx:18]: I was gonna just wait and grab it after. But i'll go ahead do that. Thanks man!

File: playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx (hunk 2 above)
Comment (jasperfurniss) [reply playbook/app/pb_kits/playbook/pb_lightbox/Carousel/index.tsx:24]: I'll update! Thanks!

**Review by nidaqg — APPROVED** [review]

**Pull request merged**